content/influxdb/clustered/reference/cli/influxctl/database/create.md

```diff
@@ -59,6 +59,15 @@ into a specified number of "buckets."
 Each of these can be used as part of the partition template.
 Be sure to follow [partitioning best practices](/influxdb/clustered/admin/custom-partitions/best-practices/).
 
+{{% note %}}
+#### Always provide a time format when using custom partitioning
+
+If defining a custom partition template for your database with any of the
+`--template-*` flags, always include the `--template-timeformat` flag with a
+time format to use in your partition template. Otherwise time will be omitted
+from the partition template and partitions won't be able to be compacted.
```

Comment:
```suggestion
time format to use in your partition template.
Otherwise, InfluxDB omits time from the partition template and won't compact partitions.
```